README.md

```diff
@@ -74,8 +74,8 @@ Note: pandoc 2.10 is short lived and 2.11 has minor API changes comparing to tha
 
 After cloning the repo and opening the panflute folder, run
 
-- `python setup.py install` to install the package locally
-- `python setup.py develop` to install locally with a symlink so changes are automatically updated
+- `python install .` to install the package locally
```

Comment:
Did you mean ``python -m pip install .``? There is no subcommand ``install`` of the ``python`` command.

Reply:
Yes. Thanks for spotting, I have fixed both references